README.md

```diff
@@ -1,19 +1,53 @@
 # ImageDePHI
+ImageDePHI is an application that can be used to redact PHI from whole slide images.
 
-This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Department of Health and Human Services, under Contract No. 75N91022C00033
 
-## To Run as a CLI
+# Installation
+ImageDePHI is under active development and as such the best way to test the latest features is to download the most current artifact.
 
-`pip install -r requirements.txt`
 
-`python imagedephi.py <directory of input images> <directory to store redacted images>`
+[ImageDePHI Artifacts](https://github.com/DigitalSlideArchive/ImageDePHI/actions/workflows/ci.yml?query=branch%3Amain)
 
+Select the most recent run:
+![ci run](imagedephi/assets/ci_run.png)
 
-## Development
+Then select the appropriate artifact for your OS:
+![artifacts](imagedephi/assets/artifacts_list.png)
 
-### Web GUI Development
-While developing the web GUI, it may be useful to maintain a running web server
-which will auto-reload code changes:
-```bash
-hypercorn --reload imagedephi.gui:app
+# Usage
+## CLI:
+Open a terminal and navigate to the directory where the artifact was downloaded.
+
+run:
+```
+./imagedephi
+```
+to get a list of cli options
+
+```
+Usage: imagedephi [OPTIONS] COMMAND [ARGS]...
+
+  Redact microscopy whole slide images.
+
+Options:
+  -r, --override-rules FILENAME  Specify user-defined rules to override
+                                 defaults
+  --help                         Show this message and exit.
+
+Commands:
+  gui   Run a web-based GUI.
+  plan  Print the redaction plan for a given image and rules.
+  run   Redact images in a folder according to given rule sets.
+```
+
+To redact a set of images supply the directory of the images to be processed and [optionally] an output directory to deposit the redacted images:
+```
+imagedephi run [path/to/images] [optional/output/path]
+```
+
+## GUI
+Alternatively you can use the gui to navigate to the desired directories:
+```
+imagedephi gui
 ```
+### **Note**: Windows users can double click the imagdephi executable icon to launch the gui outside of the terminal workflow
```

Comment:
```suggestion
### **Note**: To launch the ImageDePHI GUI, Windows users can directly run `imagdephi.exe` from Windows Explorer.
```

Reply:
fixed in a89e9c9d71a8b2ec6a0ddaa81f819ffd05d1f6e7